src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1586,6 +1586,12 @@ export function intervalFiltersDisplayValue(
         } else if (start === undefined) {
             intervalDisplayValues.push(`≤ `);
             intervalDisplayValues.push(endText);
+        } else if (
+            end === undefined &&
+            numericals[0].start === numericals[0].end
+        ) {
+            intervalDisplayValues.push(`≥ `);
+            intervalDisplayValues.push(startText);
```

Comment:
Looks better. Can we move this condition directly into `else if (end === undefined)` block?

Something like this:
```suggestion
        } else if (
            end === undefined
        ) {
            if (numericals[0].start === numericals[0].end) {
                intervalDisplayValues.push(`≥ `);
            } else {
                intervalDisplayValues.push(`> `);
            }
            
            intervalDisplayValues.push(startText);
```